WebRoot/css/zm.css

```diff
@@ -2166,6 +2166,123 @@ div.CompDragArea * {
 	overflow:hidden;
 }
 
+/* Task View style */
+.ZmTaskInfoView {
+	@TaskView@
+}
+
+.ZmReadingPaneBottom .ZmTaskInfoView {
+	@TaskViewBottomPane@
+}
+
+.ZmTaskInfoView .ZmTaskWrapper {
+	@TaskWrapper@
+}
+
+.ZmTaskInfoView .Object {
+	@Anchor@ 
+}
+
+.ZmTaskInfoView .Object-hover {
+	@Anchor-hover@
+}
+
+/* Task view header style */
+.ZmTaskHeader {
+	@TaskHeader@
+}
+
+.ZmTaskHeader > TABLE {
+	@FullWidth@
+}
+
+.ZmTaskHeader .ZmTaskHeaderRow {
+	@TaskHeaderRow@
+}
+
+.ZmTaskHeaderRow > DIV {
+	@TaskHeaderCell@
+}
+
+.ZmTaskHeaderRow .ZmTaskHeaderMain {
+	@TaskHeaderMain@
+}
+
+.ZmTaskHeaderMain .ZmTaskHeaderImage {
+	@TaskHeaderImageWrapper@
+}
+
+.ZmTaskHeaderImage IMG,
```

Comment:
Well not actually. (Side effect of ctrl +c and ctrl +v, he he)

Removed it: https://github.com/Zimbra/zm-web-client/pull/63/commits/a7b2fcdc7e6d460347cdbfc70cafc577ab173260#diff-33f9c57f22cd743824b2918132c5d73bL2215